### PR TITLE
Replace deprecated ioutil pkg + bump Go v1.19, k8s/apimachinery v0.25.2, logrus v1.9.0, open-policy-agent/opa v0.44.0, insights-plugins/plugins/opa v0.0.0-20221004151238-ef0e69a26717

### PR DESCRIPTION
**Pull request by ivanfetch-fw**
## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### What's the goal of this PR?
* Bump Go v1.17 => v1.19, k8s.io/apimachinery v0.24.3 => v0.25.2, github.com/sirupsen/logrus v1.8.1 => v1.9.0, github.com/open-policy-agent/opa v0.42.0 => v0.44.0, github.com/fairwindsops/insights-plugins/plugins/opa v0.0.0-20220613111410-770ebbca62fb => v0.0.0-20221004151238-ef0e69a26717
* Fix `go fmt` to provide helpful output in CI now that the updated CircleCI images uses a later bash shell.
* Replace deprecated use of `ioutil` pkg.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -15,16 +15,21 @@ references:
...
-      - image: circleci/golang:1.17
+      - image: cimg/go:1.19
     steps:
       - checkout
       - run:
           name: lint
           command: |
-            test -z $(go fmt ./...)
-            go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.42.1
+            fmt_output=$(go fmt ./...)
```
Comment (ivanfetch-fw) [inline .circleci/config.yml:26]: The shell changed or was updated in the new CircleCI Go image, and `test -z` returns an error when `go fat` returns more than one line.

File: cmd/insights/root.go [hunk 2]
```diff
@@ -114,7 +114,7 @@ func preRun(cmd *cobra.Command, args []string) {
 
 	configHandler, err := os.Open(configFile)
 	if err == nil {
-		configContents, err := ioutil.ReadAll(configHandler)
+		configContents, err := io.ReadAll(configHandler)
```
Comment (vitorvezani) [inline cmd/insights/root.go:117]: Cool, how did you found that `io/ioutil` was deprecated over `io`? Was it some automated tooling?

**Review by vitorvezani — APPROVED** [review]
LGTM!

File: cmd/insights/root.go (hunk 2 above)
Comment (ivanfetch-fw) [reply cmd/insights/root.go:117]: I've seen the deprecation notice in the past, in the [ioutil pkg documentation](https://pkg.go.dev/io/ioutil), but was reminded about it by [golangci-lint](https://github.com/golangci/golangci-lint) which gets run in CI.

**Pull request merged**